cmd/api/src/database/agi.go

```diff
@@ -41,9 +41,11 @@ func (s *BloodhoundDB) CreateAssetGroup(ctx context.Context, name, tag string, s
 		}
 	)
 
-	return assetGroup, s.AuditableTransaction(ctx, auditEntry, func(tx *gorm.DB) error {
+	err := s.AuditableTransaction(ctx, auditEntry, func(tx *gorm.DB) error {
 		return CheckError(tx.Create(&assetGroup))
 	})
+
+	return assetGroup, err
```

Comment:
This is a nice clean up for readability 👍 What do you think about also moving the err declaration into the var block above? The same can be done for the change on line 180 if you decide to do so.